sdk/math/storage/pool.rs

```diff
@@ -0,0 +1,27 @@
+use crate::alloc::string::ToString;
+use crate::types::{
+    fee_growth::FeeGrowth, liquidity::Liquidity, sqrt_price::SqrtPrice, token_amount::TokenAmount,
+};
+use alloc::string::String;
+
+use serde::{Deserialize, Serialize};
+use tsify::Tsify;
+use wasm_bindgen::prelude::*;
+
+#[derive(Default, PartialEq, Debug, Clone, Serialize, Deserialize, Tsify)]
+#[tsify(into_wasm_abi, from_wasm_abi)]
+pub struct Pool {
+    pub liquidity: Liquidity,
+    pub sqrt_price: SqrtPrice,
+    #[tsify(type = "bigint")]
+    pub current_tick_index: i32,
+    pub fee_growth_global_x: FeeGrowth,
+    pub fee_growth_global_y: FeeGrowth,
+    pub fee_protocol_token_x: TokenAmount,
+    pub fee_protocol_token_y: TokenAmount,
+    #[tsify(type = "bigint")]
+    pub start_timestamp: u64,
+    #[tsify(type = "bigint")]
+    pub last_timestamp: u64,
+    pub fee_receiver: String,
```

Comment:
Is there possibility to use the same struct. but changed types using `tsify` or other solution?